yamllint/rules/document_start.py

```diff
@@ -79,8 +79,12 @@
 
 ID = 'document-start'
 TYPE = 'token'
-CONF = {'present': bool}
-DEFAULT = {'present': True}
+CONF = {'present': bool,
+        'max-empty-lines-after': (int, -1),
+        'min-empty-lines-after': (int, 0)}
```

Comment:
Since `-1` and `0` are integers, why not just:
```suggestion
        'max-empty-lines-after': int,
        'min-empty-lines-after': int}
```
?

Reply:
Done.